haskell-src/Concordium/Utils.hs

```diff
@@ -23,11 +23,11 @@ class (Ixed m) => At' m where
   at' :: Index m -> Lens' m (Maybe (IxValue m))
 
 instance (Hashable k, Eq k) => At' (H.HashMap k v) where
-  at' = flip H.alterF
+  at' k f = H.alterF f k
```

Comment:
Why was this necessary or useful?

Reply:
It was necessary, did not dig into the details, might be the same/similar issue to the one Thomas pointed. (eta-expansion)